src/crimson/osd/pg.cc

```diff
@@ -907,15 +907,30 @@ void PG::mutate_object(
   }
 }
 
+void PG::enqueue_push_for_backfill(
+  const hobject_t &obj,
+  const eversion_t &v,
+  const std::vector<pg_shard_t> &peers)
+{
+  assert(recovery_handler);
+  assert(recovery_handler->backfill_state);
+  auto backfill_state = recovery_handler->backfill_state.get();
```

Comment:
assert(backfill_state)

Reply:
> assert(backfill_state)

I thought the line 916 `ssert(recovery_handler->backfill_state);` has the same effect, am I right?